source/expressions.tex

```diff
@@ -3277,29 +3277,22 @@
 ``\cvqual{cq12} \cvqual{vq12} \tcode{T}''.
 
 \item If \tcode{E2} is a (possibly overloaded) member function, function
-overload resolution\iref{over.match} is used to determine whether
-\tcode{E1.E2} refers to a static or a non-static member function.
+overload resolution\iref{over.match} is used to determine to which
+function \tcode{E1.E2} refers.
+If \tcode{E2} refers to a member function of type \tcode{T},
+\tcode{E1.E2} has type \tcode{T} and refers to that member function.
```

Comment:
The "If" here sounds like it's introducing a condition, but it's not. Perhaps:

> If `E2` is a (possibly overloaded) member function, function overload resolution ([over.match]) is used to determine to which function `E2` refers. The type of `E1.E2` is the type of `E2` and `E1.E2` refers to the function referred to by `E2`."

Reply:
Fixed with a slightly bigger hammer.